src/components/ChatLog.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import './ChatLog.css';
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+
+const ChatLog = ({entries, onUpdateChat}) => {
+    const chatEntries = entries.map((chat, index) => {
+        return (
+            <ChatEntry
+                onUpdateChat={onUpdateChat}
+                id={chat.id}
+                key={index + 1}
+                sender={chat.sender}
+                body={chat.body}
+                timeStamp={chat.timeStamp}
+                liked={chat.liked}/>  
+        )
+    })
+    return (
+        <div className="chat-messages">
+            {chatEntries}
+        </div>
+)}
```

Comment:
A more common pattern for iterating over an array of objects is to use map within the div tag (instead of using a variable to reference the return from map -- I know it's currently shown that way in Learn and it is valid, but wanted to call out the more common way to do it). See this example [here](https://github.com/Ada-C18/flasky-front-end/blob/snow-leopards/src/components/CatList.js#L10-L24)